ui/submodules/README.md

```diff
@@ -7,9 +7,14 @@
 ## dev
 
 1. `npm install` or `npm ci`
-2. `npm start` or `NU_FE_CORE_URL=http://... npm start`
+2. `npm start` or `NU_FE_CORE_URL=http://... npm start` 
 3. read logs
 
+Note: `NU_FE_CORE_URL` should point to:
```

Comment:
@mproch this is not true and is misleading

Reply:
feel free to change it to sth more correct and meaningful :) But I need some explanation in the readme, what this URL should point to...

Reply:
hmm ... I'll leave it as it is, if you know what that means, the other devs will understand as well. it's not for me - I thought that name tells everything